README.md

```diff
@@ -65,35 +65,26 @@ You can also read the MSTG on [Gitbook](https://mobile-security.gitbook.io/mobil
 
 The release 1.0 of the MSTG was published in June 2018. You can get intermediate builds in multiple formats.
 
-1. Read it on [Gitbook](https://mobile-security.gitbook.io/mobile-security-testing-guide/ "Gitbook"). The book is automatically synchronized with the main repo. You can use the gitbook command line tool to generate PDF, epub, and other e-book formats, once you have checked out this repo.
-
-To generate the epub version you can use the following command:
-
-```bash
-$ ./tools/gitbookandpdf.sh
-```
-his produces PDF, Epub and Mobi files in the "Generated" subdirectory.
+1. Read it on [Gitbook](https://mobile-security.gitbook.io/mobile-security-testing-guide/ "Gitbook"). The book is automatically synchronized with the main repo.
 
 2. Check the [releases](https://github.com/OWASP/owasp-mstg/releases "Our releases"). Here you can find a PDF, an archive containing the sources and a DocX document for any given tagged version. Please note that the documents are generated automatically per tag.
 
 3. Get the [e-book](https://leanpub.com/mobile-security-testing-guide-preview "MSTG as an e-book"). The book is available for free, but you can choose to purchase it at a price of your choosing if you wish to support our project. All funds raised through sales of the e-book go directly into the project budget and will be used to fund production of future releases.
 
-
-3. Clone the repository and run the [document generator](https://github.com/OWASP/owasp-mstg/blob/master/Tools/generate_document.sh "The document generator") (requires [pandoc](http://pandoc.org "Pandoc")). This produces docx and html files in the "Generated" subdirectory.
+3. Clone the repository and run the [document generator](https://github.com/OWASP/owasp-mstg/blob/master/Tools/generate_document.sh "The document generator") (requires [pandoc](http://pandoc.org "Pandoc")). This produces docx and HTM files in the "Generated" subdirectory.
```

Comment:
Can't check due to bad internet, but I assume it generates HTML files (<-> HTM)

Reply:
👍